woodwork/utils.py

```diff
@@ -268,17 +268,23 @@ def _is_valid_latlong_series(series):
     '''Returns True if all elements in the series contain properly formatted LatLong values,
     otherwise returns False'''
     dd = import_or_none('dask.dataframe')
+    ks = import_or_none('databricks.koalas')
     if dd and isinstance(series, dd.Series):
         series = series.compute()
-    if series.apply(_is_valid_latlong_value).all():
+    if ks and isinstance(series, ks.Series):
+        series = series.to_pandas()
+        bracket_type = list
+    else:
+        bracket_type = tuple
+    if series.apply(_is_valid_latlong_value, args=(bracket_type,)).all():
         return True
     return False
 
 
-def _is_valid_latlong_value(val):
-    '''Returns True if the value provided is a properly formatted LatLong value, otherwise
-    returns False.'''
-    if isinstance(val, tuple) and len(val) == 2:
+def _is_valid_latlong_value(val, bracket_type=tuple):
+    '''Returns True if the value provided is a properly formatted LatLong value for a
+    pandas or Dask Series, otherwise returns False.'''
```

Comment:
should it now be `for a pandas, Dask, or Koalas Series`?

Reply:
Yes, good catch!